api/dbadapters/index.ts

```diff
@@ -7,6 +7,11 @@ import { dataform } from "@dataform/protos";
 
 export type OnCancel = (handleCancel: () => void) => void;
 
+export interface IQueryResult {
+  rows: any[];
+  metadata?: any;
```

Comment:
this definitely needs a type. for now i suspect we want to do something like

```
metadata: {
  bigquery?: {
    // useful fields here
  }
}
```

that is, unless we get abstract enough fields that we don't need to namespace them by `bigquery`